sylius/resource-bundle/1.6/config/packages/sylius_resource.yaml

```diff
@@ -0,0 +1,17 @@
+# @see https://github.com/Sylius/SyliusResourceBundle/blob/master/docs/index.md
+sylius_resource:
+    # Override default settings
+    settings:
+        #paginate: null
+        #limit: null
+        #allow_paginate: [10, 20, 30]
+        #default_page_size: 10
+        #sortable: null
+        #filterable: false
+        #criteria: null
```

Comment:
There is no need to show all default config here. That is what your docs is for. 

Please remove.

Reply:
@Nyholm I just keep "settings" key to show it cause I think a lot of Sylius user doesn't know you can override them.